README.md

```diff
@@ -40,10 +64,78 @@ Fetch credentials from CyberArk Conjur using the controlling host's Conjur ident
       msg: "{{ lookup('cyberark.conjur.conjur_variable', '/path/to/secret') }}"
 ```
 
-### Author Information
-- CyberArk Business Development Technical Team 
-    - @cyberark-bizdev
-    - @enunez-cyberark
-    - @jimmyjamcabd
-- CyberArk Community and Integrations Team 
-    - @cyberark/community-and-integrations-team
+## Conjur Ansible Role
+This Ansible role provides the ability to grant Conjur machine identity to a host. Based on that identity, secrets can then be retrieved securely using the [Summon](https://github.com/cyberark/summon) tool (installed on hosts with identities created by this role).
```

Comment:
we don't even refer to the lookup plugin here. I think to tell a coherent story in the docs, we need to figure out how they are supposed to work together. I assume Summon is still an option, but we can't skip over the fact that users can also use the lookup plugin from this collection once they've used the role to create a Conjur host identity.

I'd suggest when you bump this section up higher (because you create an identity before you use it to pull secrets), you can change this to something like
```suggestion
This Ansible role provides the ability to grant Conjur machine identity to a host. Based on that identity, secrets can then be retrieved securely using the [Conjur Lookup Plugin](#conjur_variable-lookup-plugin) or using the [Summon](https://github.com/cyberark/summon) tool (installed on hosts with identities created by this role).
```